test/browser/sauce-browsers.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "browserName": "microsoftedge",
+    "platform": "WIN10",
+    "version": "20.10240"
```

Comment:
on our config?

it is behind, but it is supposed to represent the stable, beta, dev, and canary versions. if sauce only supports stable, then the default version would be ideal.